android/modules/ui/src/java/ti/modules/titanium/ui/TabGroupProxy.java

```diff
@@ -348,6 +349,30 @@ public TabProxy getActiveTab()
 		}
 	}
 
+	@Kroll.getProperty
+	public String getTitle()
+	{
+		// If the native view is drawn get the title value from the SupportActionBar
+		if (view != null) {
+			return ((AppCompatActivity) getActivity()).getSupportActionBar().getTitle().toString();
```

Comment:
We need a `null` check on:
- `getActivity()` when fetching "title" after a `TabGroup` has been closed.
- `getSupportActionBar()` when using the "Theme.AppCompat.NoTitleBar" theme.

Reply:
Can `getTitle()` return `null` if you use a `Toolbar` as an action bar without the title set?

Reply:
This turned out to be something bigger I have missed. Using a Toolbar as an ActionBar relies on adding it the usual way to a Window (through the `window.add()`), but this method is not available for the TabGroup. I am going to make setting the `supportToolbar` the only requirement for it to work for TabGroup. So far that raised a few interesting questions, but I will try to answer them all with the next update.

Reply:
Oh good point. Yeah, there is no good way to add a `Toolbar` to a `TabGroup` since we don't allow `TabGroup.add()` to add any views.

I suppose we don't "need" to add toolbar support to tab groups. Sticking to the action bar might be fine. And we have `TabbedBar` support now which (in combination with `ScrollableView`) allows app devs to construct whatever elaborate tabbing interface they want now.

Reply:
I will leave using a Toolbar as an ActionBar with TabGroup as a separate task. It turned out to be a bit more complicated than I expected. With the current way we support this for Window through Alloy and the TabGroup's implementation itself I prefer for it to be addressed together.